teuthology/util/compat.py

```diff
@@ -14,3 +14,22 @@
     from urllib import urlencode # noqa: F401
     from urllib2 import urlopen, Request, HTTPError # noqa: F401
 
+
+if PY3:
+    def stringify(s):
+        if isinstance(s, bytes):
+            return s.decode()
+        else:
+            return s
+else:
+    def stringify(s):
+        if isinstance(s, bytes):
+            return s
+        else:
+            return str(s)
+
+def bytify(b):
```

Comment:
how about using `six.ensure_binary()`?

Reply:
At the moment of this file creation we had no six module having ensure_binary method. 
Though, I was thinking about dropping six at all, and use `teuthology.util` namespace.

Reply:
But thanks for the point, I could try it. Have you tested those six.ensure* methods?

Reply:
i tried in my python3.7 env.